src/core/src/sys/cells.rs

```diff
@@ -174,6 +174,23 @@ impl UnitCell {
         return (x, y, z);
     }
 
+    /// Get a vector of the distances between faces of the unit cell
+    pub fn lengths_as_vec(&self) -> Vec<f64> {
```

Comment:
We do not need another function here, the `UnitCell::lengths` can do this too! You seems to need iteration on the lengths, so why not change the signature to be `pub fn lengths(&self) -> [f64; 3]`? This way you get iteration, and do not pay for a dynamic allocation.

The current function is only [used once](https://github.com/lumol-org/lumol/blob/677d697b91ba41f0ef6c578968cf1ad8d551df21/src/core/src/energy/global/ewald.rs#L124-L134), for the exact same reason as here.